client/src/index.jsx

```diff
@@ -1,8 +1,9 @@
 import React, {useState} from 'react';
 import ReactDOM from 'react-dom';
-//import API_KEY from './config.js';
 import axios from 'axios';
-import getStartingPid from './getStartingPid.jsx'
+import Overview from './overview/index.jsx'
```

Comment:
you _should_ just be able to import from `"./overview" and the assumed file will be `index`

also! take a look at [absolute pathing](https://betterprogramming.pub/use-absolute-paths-with-react-51ced66f119f) in case you find yourself creeping more deeply into the folders you're looking for